onnxruntime/core/providers/cpu/tensor/cast_op.cc

```diff
@@ -30,8 +30,9 @@ using namespace boost::mp11;
 namespace onnxruntime {
 
 namespace op_kernel_type_control {
+// we're using one set of types for all opsets of Cast, so use -1 as a generic opset value
```

Comment:
would it be useful to have -1 as a constant for a generic opset? i suppose any value would work, but that may help with consistency in the future.